repos/system_upgrade/el7toel8/actors/setpermissiveselinux/actor.py

```diff
@@ -16,17 +16,12 @@ class SetPermissiveSelinux(Actor):
 
     name = 'set_permissive_se_linux'
     consumes = (SelinuxPermissiveDecision,)
-    produces = (FinalReport,)
+    produces = ()
     tags = (FinalizationPhaseTag, IPUWorkflowTag)
 
     def process(self):
         for decision in self.consume(SelinuxPermissiveDecision):
             if decision.set_permissive:
                 success, err_msg = selinux_set_permissive()
                 if not success:
-                    self.produce(FinalReport(
```

Comment:
Same here, why is this removed? The whole point was that we can do the post install reports too. No?

Reply:
It is nothing against the final report. However taking a look back into it this shoulds be something to keep reporting indeed. Will get it back. Applies also for the previous one.